migrations/src/test/resources/application.properties

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
 
-za.co.absa.enceladus.menas.mongo.connection.string=mongodb://localhost:27017
+menas.mongo.connection.string=mongodb://localhost:27017
 za.co.absa.enceladus.migrations.mongo.connection.database=migrations_integration
```

Comment:
For migrations the config keys might become `enceladus.*` (e.g. `enceladus.migrations.mongo.connection.database`), right?
(with `za.co.absa` removed)

Reply:
I think even without `enceladus`

Reply:
Cool, also need to change `IntegrationTestConfiguration.scala` then. It has:
```
  private val connectionStringKey = "za.co.absa.enceladus.menas.mongo.connection.string"
  private val databaseKey = "za.co.absa.enceladus.migrations.mongo.connection.database"
```

Reply:
These are the test I have talked about

Reply:
I see